packages/nocodb-sdk-v2/src/sdk/base.ts

```
@@ -0,0 +1,18 @@
+import { Base } from './lib/Api';
+
+export class NocoDBBase extends Base {
+  readonly id: string;
+  private workspace: Workspace;
+  private readonly internalAPI: InternalAPI;
+
+  constructor(internalApi: InternalAPI, id: string, workspace: Workspace) {
+    this.internalAPI = internalApi;
+    this.id = id;
+    this.workspace = workspace;
+  }
```

Biome (1.9.4)

[error] This constructor has code paths accessing `this` without calling `super()` first.

`this` is accessed here:

If this is intentional, add an explicit throw statement in unsupported paths.

(lint/correctness/noUnreachableSuper)